### DISCUSSION TITLE
Do we have a map of all the available pins on the 5m and Pro motherboards?

### DISCUSSION THREAD

**Discussion by johncotten** [xblax/flashforge_ad5m_klipper_mod#85] [Q&A] [answered]
I am wondering if we want to add additional devices if we know of empty pins we could use to trigger those devices?
One I am thinking is maybe adding a chamber heater.

**Comment by consp** [comment, marked as answer]
A quick look at the pcb for the pro board doesn't seem there is any (MOS)FETs left which can be triggered autonomously (easilly). Though there are two more USB ports and a UART which is free so options abound in those cases.
